R/opts.R

```diff
@@ -446,6 +446,15 @@ gp_opts <- function(basis_prop = 0.2,
 #' empty a mean (`mean`) and standard deviation (`sd`) needs to be supplied
 #' defining the normally distributed scaling factor.
 #'
+#' @param na Character. Options are "missing" (the default) and "accumulate".
+#'   This determines how NA values in the data are interpreted. If set to
```

Comment:
Sorry this is so boring but indents here are not in line with the rest of this doc....

Reply:
😲 blame Emacs. Will update.

Reply:
Actually this seems to be what roxygen suggests, and it does make the code more readable.
https://mpn.metworx.com/packages/roxygen2/7.1.0/articles/rd.html#object-specifics
Will change all of the ones here to be consistent.